### DISCUSSION TITLE
App: use disaplay template fields as a supplementary query fields in relateion interface

### DISCUSSION THREAD

**Discussion by everyx** [directus/directus#2706] [Feature Requests]
I want show `product_property_items_id.property.name` and `product_property_items_id.value` so I set this in this `many to many` relation, but is empty when I view product item 

![image](https://user-images.githubusercontent.com/1665522/81777014-8c178600-9522-11ea-9aa6-38cf0cdd1ccb.png)

![image](https://user-images.githubusercontent.com/1665522/81776902-4bb80800-9522-11ea-92b2-15f38fda4d7a.png)

![image](https://user-images.githubusercontent.com/1665522/81776816-1e6b5a00-9522-11ea-9f7d-2af9a025af93.png)

apps should consider adding flexibility in this case.

**Comment by rijkvanzanten** [comment]
Yes! Yes! Yes!

As a matter of fact, I actually just implemented this and got it working in the v9 in works branch 😄 

The display template for m2o can go indefinitely deep in v9+ 💪
